CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.10.0]
+### Added
```

Comment:
I think we should also add information on the `water_mask.py` functionality that has been remove with the switch from a polygon to raster approach. Could you add some text to this effect?

Reply:
I'm don't think any functionality has been removed. The inputs and outputs are the same. The water masks were fed to isce2 as rasters previously. Could you please clarify what functionality you are referring to?